bluez-async/src/device.rs

```diff
@@ -296,6 +307,12 @@ mod tests {
         assert_eq!(device_id.to_string(), "hci0/dev_11_22_33_44_55_66");
     }
 
+    #[test]
+    fn from_string() {
+        let device_id = DeviceId::try_from("hci0/dev_11_22_33_44_55_66");
+        assert_eq!(device_id.unwrap().to_string(), "hci0/dev_11_22_33_44_55_66");
+    }
```

Comment:
Please add a test for converting an invalid path String to a `DeviceId` too, to make sure it returns an error as expected.